src/contexts/nftRewardsContext.ts

```diff
@@ -14,6 +14,7 @@ export const NftRewardsContext = createContext<NftRewardsContextType>({
     CIDs: undefined,
     rewardTiers: undefined,
     postPayModal: undefined,
+    delegateVersion: undefined,
```

Comment:
hmmm i wonder if `contractVersion` is clearer here. In the context of NftRewards, seems like it might make sense.